docs/articles/baseplatform/booting-qemu-image.rst

```diff
@@ -61,14 +113,29 @@ KVM option    Value           Description
 Connecting to the virtual machine using SSH
 -------------------------------------------
 
-The machine can be accessed via SSH for development purposes using the
-following command:
+The machine can be accessed via SSH for development purposes.
+
+SSH access via Port forwarding:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Boot the qemu machine with port forwarding configuration, and then run the
+following command on host machine.
 
 .. code-block:: bash
 
     $ ssh root@127.0.0.1 -p 1234
```

Comment:
Can we add an explanation for this so that users understand that 1234 map to 22?